integrationTests/vm/txsFee/multiShard/relayedTxScCalls_test.go

```diff
@@ -104,7 +104,7 @@ func TestRelayedTxScCallMultiShardShouldWork(t *testing.T) {
 
 	txs := testContextInnerSource.GetIntermediateTransactions(t)
 
-	intermediateTxs = testContextInnerSource.GetIntermediateTransactions(t)
+	_ = testContextInnerSource.GetIntermediateTransactions(t)
```

Comment:
check here as well if the second call is needed

Reply:
one get is enough